### ISSUE TITLE
Replacing/hiding database search message with a visual element

### ISSUE THREAD

**Issue by eriolloan** [D0-0K/adwaitapod#27]
**Device**: iPod 7th gen
**Rockbox Version**: Dev build from X month, or Release 3.15 etc.
**Adwaitapod Theme**: all.

Hi, not an issue per say, maybe more of a feature request.
Your theme is so clean that de style difference with rockbox messages is glaring.

I don't think fixing this is in the scope of a theme, but do you think there would be a way to theme the "searching" notification when navigating the database ?

Maybe something along those lines :

![adwaitapod-search_mockup](https://github.com/D0-0K/adwaitapod/assets/15250783/1ff5c044-adfb-444d-a22d-3177513bfc2e)

By the way, thanks for pushing the boundaries with your themes. If rockbox brought new life to my device, your themes made me fall in love with it again :D

**Comment by D0-0K** [comment]
Hi there!

 Thanks for the feature request, I _really_ like this idea! The database feature isn't something I use too much, so this notification dialog has never caught my attention. There wouldn't be a direct way of doing so, but I can think of a few things I could try. The Simulator is funny enough too fast to see this page, and I'm currently repairing my iPod, but once it's running again I can take a look at implementing this properly.

Either way, I can definitely see this being a part of the next major update! (love the icon you made up respecting the adwaita icon theme by the way!) And I'm very glad to hear that my themes are making your iPod more usable!! :)

**Comment by eriolloan** [comment]
My you're fast !

That's so cool you're interested by this and I'm thrilled you'll take a shot at it.

The message doe's not display very long in my case, having switched to SD card, but I know from experience that's something you see ALOT when using the stock disk drive.

Thank for the feedback on my quick icon. Here's the mockup's file if you want.

![adwaitapod-search_mockup](https://github.com/D0-0K/adwaitapod/assets/15250783/ad153c9e-42ef-4226-952e-8e6bc623f0ee)

Have a great day, and thank you for your work.
Btw, is there a way to tip you ?

**Comment by D0-0K** [comment]
Yeah, the database initialisation also takes a noticeable amount of time on hard drive as well so that might be work theming perhaps too. 

Thinking about it some more, the one thing I do know is that the rectangular messages can't be disabled (understandably though), but I'm hoping to get some time to contribute to the Rockbox core in the future and I think finding a suitable way to theme these dialogs could be interesting.

Thanks, that mockup file will be very helpful! I won't be working on this fully until after I update [Themify](https://github.com/D0-0K/themify), but I'll be sure to update you here :)

And thank you for the tip! It means so much to receive support!
